tox.ini

```diff
@@ -1,7 +1,27 @@
 [tox]
-envlist = py27, py35
+envlist = py27, py35, py37, flake8, docs, check-manifest
```

Comment:
Is `py36` excluded deliberately?

Reply:
No, might be worth adding py36 as well.
On the other hand, testing all python versions might be overkill, tox support ``py3`` specifier which use version of the "python3" binary. On travis side we can test explicitly versions we want.

Reply:
Just keep testing py3.5 as well as py3.7 I suppose. We can skip py3.6, but I wan to know if something blows up in 3.5 or below because `anosql` is committed to maintaining support for it at this time (at least for this version). We would want to know if somebody accidentally used an `f""` string and broke py3.5.

Thanks for doing this, I hadn't used tox until I started helping with this project so I am by no means an expert at setting it up. I'm glad to have your help.